extensions/stats/config.proto

```diff
@@ -68,6 +68,7 @@ message PluginConfig {
   // next id: 7
   // The following settings should be rarely used.
   // Enable debug for this filter.
+  // DEPRECATED.
```

Comment:
hide from docs?

Reply:
I think we should keep it to reduce confusion for now. Deprecated fields must stay until we can actually remove usages.